docs/dev/apis/lethallib/custom-enemies/blender-resources.md

```diff
@@ -0,0 +1,103 @@
+:::
+
+- [The Nuts and Bolts of Blender's animation system](https://youtu.be/p3m57yAcsi0) - CGDive
+    - Introduces concepts in a very in-depth way. Introduces Timeline, Dope Sheet, Graph Editor, NLA Editor, Actions.
+- [Un-confusing the NLA Editor (Nonlinear Animation)](https://youtu.be/tAo7HxxxA08) - GCDive
+    - A more in-depth video about the NLA Editor. Do note though, we do not need to do anything complex with the NLA Editor.
+- [Become a PRO at Animation in 25 Minutes | Blender Tutorial](https://youtu.be/_C2ClFO3FAY) - CG Geek
+    - Animating a walk cycle. Uses Timeline, Dope Sheet and Graph Editor. Uses references for animation.
+- [Character animation for impatient people - Blender Tutorial](https://youtu.be/GAIZkIfXXjQ) - Joey Carlino
+    - If you don't want to make and rig your own models.
+
+### Animation - Common Issues
+
+**Objects in my model appear in different places as in Blender when exporting to Unity.**  
+This might be because you have directly animated an object, instead of an armature. Try parenting your object to an armature and remake your animation with that.
+
+**Animations are broken in Unity.**  
+This might be because you have animations with the same name in your NLA editor. Make sure your animations have unique names.
+
+## Exporting Assets For Unity
+
+To export your model, go to: `File` -> `Export` -> `FBX (.fbx)`  
+This will open our FBX exporter window, where we have some options available to us. If you have put your animations in the NLA editor, you'll want to disable `All Actions` under the `Bake Animation` dropdown.
+
+![Screenshot: Export as FBX Settings](/images/lethallib/custom-enemies/blender/BlenderExportAsFBX.png)
+
+The most important thing here however is the transform section. Because of the differences in Blender's and Unity's coordinate systems, exporting your model is not quite as straight-forward as you'd think. It's very easy to get your model pointing in the wrong direction, being sideways, or even upside down if you don't have correct values set.
+
+Blender considers "Forward" to be `-Y`, while in Unity it's `Z`. And "Up" in Blender is `Z`, while in Unity it's `Y`. "Forward" is illustrated with the green arrow in the image below, while the blue arrow is "Up". The left side represents Blender's coordinate space, center represents our export settings in Blender's space, while the right side represents the output in Unity. We set `Forward` (`-Y`) to `-Z Forward`, and `Up` (`Z`) to `Y Up` and our model appears correctly in Unity space. The reason `-Z Forward` becomes positive `Z` might be because technically, `Z` is inverted between Blender and Unity, which we can see in the image.  
```

Comment:
Consider splitting this paragraph into two lines to make it easier to read (you just need to add two new lines at somepoint in the paragraph)

Reply:
I moved a part of the text below the image instead.